automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationPdSpec.scala

```diff
@@ -0,0 +1,113 @@
+
+  "create and attach a persistent disk" in { googleProject =>
+    val diskName = genDiskName.sample.get
+    val diskSize = genDiskSize.sample.get
+    val runtimeName = randomClusterName
+    val createRuntimeRequest = defaultCreateRuntime2Request.copy(
+      runtimeConfig = Some(
+        RuntimeConfigRequest.GceWithPdConfig(
+          None,
+          PersistentDiskRequest(
+            diskName,
+            Some(diskSize),
+            None,
+            None,
+            Map.empty
+          )
+        )
+      )
+    )
+
+    // validate disk still exists after runtime is deleted
+    val res = dependencies.use { dep =>
+      implicit val client = dep.httpClient
+      for {
+        _ <- LeonardoApiClient.createRuntimeWithWait(googleProject, runtimeName, createRuntimeRequest)
+        _ <- LeonardoApiClient.deleteRuntimeWithWait(googleProject, runtimeName)
+        disk <- LeonardoApiClient.getDisk(googleProject, diskName)
+        _ <- LeonardoApiClient.deleteDiskWithWait(googleProject, diskName)
+        diskAfterDelete <- LeonardoApiClient.getDisk(googleProject, diskName)
+      } yield {
+        disk.status shouldBe DiskStatus.Ready
+        disk.size shouldBe diskSize
+        diskAfterDelete.status shouldBe DiskStatus.Deleted
+      }
+    }
+    res.unsafeRunSync()
+  }
+
+  "create and attach an existing a persistent disk" in { googleProject =>
+    val runtimeName = randomClusterName
+    val diskName = genDiskName.sample.get
+    val diskSize = genDiskSize.sample.get
+
+    val res = dependencies.use { dep =>
+      implicit val client = dep.httpClient
+
+      val createRuntimeRequest = defaultCreateRuntime2Request.copy(
+        runtimeConfig = Some(
+          RuntimeConfigRequest.GceWithPdConfig(
+            None,
+            PersistentDiskRequest(
+              diskName,
+              Some(DiskSize(500)), //this will be ignored since in this test we'll pre create a disk
+              None,
+              None,
+              Map.empty
+            )
+          )
+        )
+      )
+
+      for {
+        _ <- LeonardoApiClient.createDiskWithWait(googleProject,
+                                                  diskName,
+                                                  defaultCreateDiskRequest.copy(size = Some(diskSize)))
+        _ <- createRuntimeWithWait(googleProject, runtimeName, createRuntimeRequest)
+        runtime <- getRuntime(googleProject, runtimeName)
+        _ <- deleteRuntimeWithWait(googleProject, runtimeName)
+        runtimeAfterDeletion = org.broadinstitute.dsde.workbench.leonardo.Leonardo.cluster
+          .listIncludingDeletedRuntime(googleProject)(ronAuthToken)
+        _ = println(s"runtimeAfterDeletion ${runtimeAfterDeletion}")
+        _ <- deleteDiskWithWait(googleProject, diskName)
+      } yield {
+        runtime.diskConfig.map(_.name) shouldBe Some(diskName)
+        runtime.diskConfig.map(_.size) shouldBe Some(diskSize)
+      }
+    }
+
+    res.unsafeRunSync()
+  }
+}
+
```

Comment:
I don't want to compile more automation tests on this PR since it's already pretty big...But yea, I think Gabriella will be adding that test since we can't test it until her change